src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1765,43 +1770,34 @@ private static string FormatErrorMessage(string error, string contentType)
         }
 
         // Returns true if the status code is one of the supported redirection codes.
-        private static bool IsRedirectCode(HttpStatusCode code)
+        private static bool IsRedirectCode(HttpStatusCode statusCode) => statusCode switch
         {
-            int intCode = (int)code;
-            return
-            (
-                (intCode >= 300 && intCode < 304) ||
-                intCode == 307 ||
-                intCode == 308
-            );
-        }
+            HttpStatusCode.Found or
+            HttpStatusCode.Moved or
+            HttpStatusCode.MultipleChoices or
+            HttpStatusCode.PermanentRedirect or
+            HttpStatusCode.SeeOther or
+            HttpStatusCode.TemporaryRedirect => true,
+            _ => false
```

Comment:
Apply indentation to make it easier to view the switch clauses.
```suggestion
            HttpStatusCode.Found or
                HttpStatusCode.Moved or
                HttpStatusCode.MultipleChoices or
                HttpStatusCode.PermanentRedirect or
                HttpStatusCode.SeeOther or
                HttpStatusCode.TemporaryRedirect => true,
            _ => false
```

Reply:
I moved the `or` instead, I think it's better